go/tendermint/genesis_single.go

```diff
@@ -31,7 +36,30 @@ func (p *singleNodeGenesisProvider) GetTendermintGenesisDocument() (*tmtypes.Gen
 // running a single node "network", primarily for testing.
 func NewSingleNodeGenesisProvider(identity *identity.Identity) (genesis.Provider, error) {
 	doc := &genesis.Document{
-		Time: time.Now(),
+		Time:             time.Now(),
+		ConsensusBackend: "tendermint",
+		EpochTime: epochtime.Genesis{
+			Backend: "tendermint_mock",
```

Comment:
Whowa, really?  Yes, this is used by the unit tests.  It also was used when the node was being brought up stand-alone.  If we no longer care to support that use case, we should make this code significantly harder to trigger.

Reply:
I'll implement the suggestion from comment https://github.com/oasislabs/oasis-core/pull/2294#discussion_r340552075.